### PR TITLE
desc for deprecated software

### PR DESCRIPTION
**Pull request by antgonza**

### CODE REVIEW

File: qiita_pet/support_files/doc/source/processingdata/qp-fastp-minimap2.rst [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+was not expected by us. It was discovered when manually assessing replicated
+sequences, which on examination by BLAST against NT reported to be adapters.
+
+Adapter filtering for paired-end data with fastp requires specifying either the
+exact adapters to remove (i.e., no autodetection), or to explicitly specify “--detect_adapter_for_pe”. Qiita previously indicated to users that the
+qp-fastp-minimap2 plugin was performing adapter autodetection and filtering.
+However, because this flag was not specified, that behavior did not occur.
+
+In the metagenomic dataset the adapters were discovered in, we observed a few
+sequences with high replication, which assignments to a few genomes in RS210.
```
Comment (wasade) [inline qiita_pet/support_files/doc/source/processingdata/qp-fastp-minimap2.rst:17]:
```suggestion
sequences with high replication, with assignments to a few genomes in RS210.
```

**Comment by wasade** [comment]
Awesome thanks!

File: qiita_pet/support_files/doc/source/processingdata/qp-fastp-minimap2.rst [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+At the end of August 2023, we discovered that the parameters used by
+qp-fastp-minimap2 did not trigger application of adapter filtering. By default,
+fastp performs autodetection of adapters and filtering for single-end data. By
+default, fastp does not perform these operations on paired-end data. This behavior
+was not expected by us. It was discovered when manually assessing replicated
+sequences, which on examination by BLAST against NT reported to be adapters.
+
+Adapter filtering for paired-end data with fastp requires specifying either the
+exact adapters to remove (i.e., no autodetection), or to explicitly specify “--detect_adapter_for_pe”. Qiita previously indicated to users that the
```
Comment (charles-cowart) [inline qiita_pet/support_files/doc/source/processingdata/qp-fastp-minimap2.rst:12]: This line appears to run longer than the others. I think it's meant to break after 'specify'?

**Review by charles-cowart — CHANGES_REQUESTED** [review]
Everything looks good! I just have minor suggestion about adding a line break.

File: qiita_pet/support_files/doc/source/processingdata/qp-fastp-minimap2.rst (hunk 2 above)
Comment (antgonza) [reply qiita_pet/support_files/doc/source/processingdata/qp-fastp-minimap2.rst:12]: Agreed but that's merely a non written or enforced rule ... I think is fine but if you feel strongly about it I can add a new line.

Comment (charles-cowart) [reply qiita_pet/support_files/doc/source/processingdata/qp-fastp-minimap2.rst:12]: Nope! no worries on my end.

**Review by charles-cowart — APPROVED** [review]

**Pull request merged**